### DISCUSSION TITLE
Negative internal energy encountered during the calculation process.

### DISCUSSION THREAD

**Discussion by RachelMeng0096** [AMReX-Combustion/PeleC#735] [Q&A] [answered]
Greetings,  I have set up a jet in crossflow simulation based on the cavityflame example. The results appear normal thus far. However, I've encountered instances of negative eint_e during computations. I'm curious about the potential causes of this anomaly. Could it impact the accuracy of my results? I would appreciate your insights on this matter.
![image](https://github.com/AMReX-Combustion/PeleC/assets/153271449/26b0f004-a68a-447c-b116-301ee3b9983b)

**Comment by hsitaram** [comment, marked as answer]
rho*e in pelec includes both sensible and chemical energy. So it can be negative depending on species formation energies.
